Makefile

```diff
@@ -46,6 +48,13 @@ cargo:
 clean:
 	rm -rf target/
 
+clientgen:
+	docker run --rm -v "$(shell pwd):/local" --user "$(shell id -u):$(shell id -g)" openapitools/openapi-generator-cli generate \
```

Comment:
Why are you doing this in a docker container? And, where are the openapitools listed as a prequisite?

Reply:
If this was not done in a container you would need nodejs and npm/yarn installed in order to build the CLI, since we're not checking in the client files at the moment.  Could do that, but this is how the other projects also generate their clients.  This pulls the container from Docker Hub so there's not much in the way of prerequisites other than being able to run docker.  We may choose to check in the client files like we are in other projects, but for now I left it on the side to get us started.